twilight-gateway/src/shard.rs

```diff
@@ -668,6 +668,11 @@ impl Shard {
             ratelimiter.acquire_one().await;
         }
 
+        self.send_direct(message).await
+    }
+
+    /// Send a raw websocket message without first passing the ratelimiter.
+    async fn send_direct(&mut self, message: Message) -> Result<(), SendError> {
```

Comment:
this name would be more clear:
```suggestion
    async fn send_unratelimited(&mut self, message: Message) -> Result<(), SendError> {
```